### PR TITLE
Enable pipeline tests

### PR DESCRIPTION
**Pull request by mtojek**
## What does this PR do?

This PR enables pipeline tests in the repository using the builder tool `elastic-package`. It spins up the `elasticsearch` instance, executes tests and tears it down.

First candidate is the `nginx` integration.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/CONTRIBUTING.md#tips-for-building-integrations) and this pull request is aligned with them.
- [x] I have verified that all datasets collect metrics or logs.

## Author's Checklist

- [ ]

## How to test this PR locally

`mage check`

## Related issues

-

### CODE REVIEW

File: magefile.go [hunk 1]
```diff
@@ -216,9 +217,75 @@ func ModTidy() error {
 	return nil
 }
 
+func Test() {
+	defer tearDownStack()
```
Comment (ycombinator) [inline magefile.go:221]: Should this be here or in `bootUpStackElasticsearch`, since that's where the stack is spun up?

File: magefile.go [hunk 2]
```diff
@@ -237,10 +304,10 @@ func runElasticPackageOnAllIntegrations(subCommand string) error {
 			return errors.Wrapf(err, "chdir failed (path: %s)", packagePath)
 		}
 
-		fmt.Printf("%s: elastic-package %s\n", packagePath, subCommand)
-		err = sh.Run(filepath.Join(workDir, "build", "elastic-package"), subCommand)
+		fmt.Printf("%s: elastic-package %s\n", packagePath, strings.Join(subCommandWithArgs, " "))
+		err = sh.RunV(filepath.Join(workDir, "build", "elastic-package"), subCommandWithArgs...)
```
Comment (ycombinator) [inline magefile.go:308]: Nit: Now that you've defined `runElasticPackage`, should we call it from here?

File: magefile.go (hunk 1 above)
Comment (mtojek) [reply magefile.go]: Fixed

File: magefile.go (hunk 2 above)
Comment (mtojek) [reply magefile.go]: It was hard a bit to refactor it (due to jumping over directories), but I think it works now.

**Pull request merged**